Sources/Classes/core/Protocols/MiniAppBridgeModel.swift

```diff
@@ -0,0 +1,5 @@
+/// Struct is used to store the String content recieved from the Miniapps
+public struct MiniAppBridgeContent {
```

Comment:
Same comment here as well

Reply:
I have now updated the name to `UniversalBridgeContent`